### ISSUE TITLE
Wrong VAT amount

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#5412]
## Description

For a particular order on French prod: order number R767327431

`/admin/reports/sales_tax` is showing an amount of VAT at 2,1% of 0,05 cents.

Yet this hub has no product at 2,1% VAT and the order isn't showing any line with a 0,05 euros fee.

I searched through payment method or shipping method taxes... but I can't find anything...

## Expected Behavior

`/admin/reports/sales_tax` shouldn't show a positive VAT amount for the 2,1% tax for hub 922.

## Actual Behaviour

A VAT amount at 2,1% is shown by the report for this order.

## Steps to Reproduce

I don't know

## Severity
bug-s3: a feature is broken but there is a workaround

## Your Environment
* Version used: v2.9.8
* Browser name and version: Firefox
* Operating System and version (desktop or mobile): Desktop

**Comment by luisramos0** [comment]
I see 5cents as included tax of this (manual?) adjustment:
![image](https://user-images.githubusercontent.com/1640378/81730257-29bc7800-9485-11ea-9e83-a8762ac45014.png)

I am not sure what value is these 5cents, but if 0.05 is tax included in 1.83 than the tax is something like 3,4%
I am not sure...

Another idea: 2.1% is the super Reduit tax rate maybe applicable to a product that is no longer in the order?

**Comment by RachL** [comment]
> Another idea: 2.1% is the super Reduit tax rate maybe applicable to a product that is no longer in the order?
Ah yes thanks! I remember that you gave me this lead now on Slack. I've explored the possibility, but the product should still be in the catalog with the wrong tax right? Or they would have deleted both the line item and the product... and only on that order. 🤯 

> I am not sure what value is these 5cents, but if 0.05 is tax included in 1.83 than the tax is something like 3,4%

Yes and actually only the total tax value of the /admin/reports/sales_tax shows the tax _with_ adjustments. So all the amounts oh the reports per tax rate are _without_ adjustments.

That being said, I have no idea how this 0.05 can come out of 1.83. So I went to edit the adjustments and I have this weird message asking me to check the right tax rate amount (I didn't hit save, just edit):

![image](https://user-images.githubusercontent.com/6525576/81732688-216a3a00-9491-11ea-99f1-19bd2e2f974f.png)

I don't know why I never tried to edit the adjustment before. I will ask them if we can re-save the adjustment and let you know if this has any effect on the report. Thanks!!

**Comment by RachL** [comment]
Ok thank you @luisramos0 I think I have traced what happened: they selected the wrong tax rate for the adjustment. When editing the tax rate as the display is not correct they assumed that they didn't do a mistake.
I've opened: https://github.com/openfoodfoundation/openfoodnetwork/issues/5422 for the display problem.
And I can't wait for us to get invoices with all history that happened to an order!! 
Closing this one
